docs/architecture/_index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Viam Architecture"
+linkTitle: "Architecture"
+weight: 409
+type: "docs"
+description: "Overview of how a machine running on Viam is structured."
+imageAlt: "Viam architecture"
+images: ["/get-started/installation/thumbnails/configure.svg"]
+tags: ["components", "services", "communication"]
+menuindent: true
+---
+
+![Viam data, ML and registry in the cloud and Viam fleet and build on-premises. Viam core is the communication layer that connects the cloud to the on-device portion.](/architecture/cloud-premises.png)
```

Comment:
I think possibly the one's you have below are a good fit. I don't know that we need more

Reply:
So delete this and replace with an intro sentence?